src/vision.cs

```diff
@@ -1,4 +1,4 @@
-﻿//
+//
```

Comment:
Dunno what happened here :P

Reply:
Oh I changed from UTF8 BOM to just plain UTF8

Reply:
Scandalo!